### PR TITLE
[Issue 4044][docs] Fix outdated docker-compose advertised address

### LINKED ISSUE #4044: Pulsar Dashboard Displays Very Little Information
**Describe the bug**

The pulsar-dashboard does not display the information published by the pulsar cluster.

**To Reproduce**
1. Install a [bare metal pulsar](https://pulsar.apache.org/docs/en/deploy-bare-metal/)
    1. on an openstack cloud
    2. rather than a DNS server distribute identical /etc/hosts files to each of the vm's and the development machine
    2. 3 zookeeper each on their own vm
    3. 3 bookie/broker each pair on their own vm
2. On the development machine: Git [clone the pulsar](https://github.com/apache/pulsar/tree/v2.3.0)
    1. build the docker image : docker build -t apachepulsar/pulsar-dashboard dashboard
    2: export SERVICE_URL=http://ledge2:8080/
    3: docker run -p 80:80   -e SERVICE_URL=$SERVICE_URL   apachepulsar/pulsar-dashboard
3. Construct a 
    1. single tenant 
    2. several namespaces
    3. several topics

**Expected behavior**
The pulsar-dashboard should show clusters, tenants, namespaces, etc.

**Screenshots**
The pulsar  http server shows the presence of the cluster ...
image(2)

The pulsar-dashboard apparently does not know about the cluster...
image(1)

But no if you bring up one of the tabs having a CLUSTERS menu on the right there is the missing cluster...
image

**Desktop (please complete the following information):**
 - OS: Ubuntu 18.04 for the development machine and vm's

**Additional context**
Because /etc/hosts is used instead of DNS it is necessary to use the ledger2 ipaddr.
```
 export SERVICE_URL=http://128.60.107.171:8080/ 
```

### PR DESCRIPTION
**Pull request by bithavoc**
Fixes #4044 and #3367, completes streamnative/pulsar/issues/290

### Motivation

When you google "docker compose pulsar", the first results points to `docker-compose/standalone-dashboard/docker-compose.yml` which works for the most part if you use CLI commands such as `pulsar-admin`, but the dashboard never shows any tenant, topics, etc. This has been reported in issues such as #4044 and officially given a recommendation in streamnative/pulsar/issues/290, however, `docker-compose/standalone-dashboard/docker-compose.yml` is still outdated.

### Modifications

- Modified `docker-compose/standalone-dashboard/docker-compose.yml` to include `--advertised-address` with the name of the network host inside the docker-compose network that the dashboard can actually use.

### Verifying this change

This change is a trivial rework / code cleanup without any test coverage.

This change added tests and can be verified as follows:

  - without applying this patch, run `docker-compose up`, even after 2 minutes the dashboard still doesn't show tenants or topics
  - apply the patch and then run `docker-compose up`, refresh the dashboard UI after 20 seconds and the tenants and topics show up.

### CODE REVIEW

**Review by sijie — APPROVED** [review]

**Comment by bithavoc** [comment]
@sijie I see this PR is assigned to me but I'm having a hard time what's the next step from my side, I'm looking at the CONTRIBUTING guide and it states:

> Commit: Once at least 2 Pulsar committers have approved the pull request, a Pulsar committer will merge it into the master branch (and potentially backport to stable branches in case of bug fixes).

So I guess I need another commiter to approve? Also the CI is complaining about something but I can't really tell what's the issue is, I can't seem retry the action either and the action step doesn't load the log output(somehow private?).

It's my first contribution to the project, can you give me some clues on how to proceed?

**Comment by sijie** [comment]
@bithavoc I assigned it to you to grant the contribution credit to you. We probably just need approval from another committer and then we can merge this pull request.

The CI failures are not unexpected. There is some flakiness on the CI. I will rerun those CI tests. You don't need to worry about it.

**Review by merlimat — APPROVED** [review]

**Pull request merged**
